portfolio/src/main/webapp/index.html

```diff
@@ -4,18 +4,55 @@
     <meta charset="UTF-8">
     <title>Andrew Pham | Home</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js">
-        
-    </script>
+   
   </head>
   <body>
     <div id="content">
-      <h1> Andrew Portfolio</h1>
+      <h1>Andrew Portfolio</h1>
       <h3>Hello! My name is Andrew Pham and I am a second year at the University of California San Diego. I grew up with a family of six: two parents, an older brother and sister, <br>
             and a younger brother. My favorite color is purple and the type of music I like to listen to is Kchill and lofi.<h3>
       <p>This is my portfolio.</p>
     </div>
 
+    <br> <br> <br> <br>
+
+        <button type="button" class="collapse" style="background-color: #ea4f5e;">Bolbbalgan4</button>
```

Comment:
I'd ask Scott about this, but he recommended that I don't put "style=" in my html tags and instead move that into a css file (this would apply for all of your buttons, but I'm just going to make this one comment)